webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -649,6 +953,9 @@ class NaiveNodeInfoServiceCachedImpl(
     }
   }
 
+  override def getBackendLdapNodeInfo(nodeIds: Seq[String]): IOResult[Seq[LDAPNodeInfo]] = ???
```

Comment:
this breaks com.normation.rudder.services.queries.TestQueryProcessor
with
An unchecked error was produced.
scala.NotImplementedError: an implementation is missing
	at scala.Predef$.$qmark$qmark$qmark(Predef.scala:347)
	at com.normation.rudder.services.nodes.NaiveNodeInfoServiceCachedImpl.getBackendLdapNodeInfo(NodeInfoService.scala:956)
	at com.normation.rudder.services.nodes.NodeInfoServiceCached.$anonfun$withUpToDateCache$12(NodeInfoService.scala:644)
	at zio.internal.FiberContext.evaluateNow(FiberContext.scala:815)